webstore-app/overlays/qa/cms-content-cm.yaml

```diff
@@ -0,0 +1,14 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: cms-content-cm
+  annotations:
+    reloader.stakater.com/match: "true"
+data:
+  VC_PLATFORM_MODULES_CONFIG: https://raw.githubusercontent.com/VirtoCommerce/vc-deploy-apps/qa/webstore-app/overlays/qa/modules.json?8439F69CD65989A8EDE4F37181261D2E9964E81DB0F94742F9FE82C4830AC883
```

Comment:
Modules is not CMS content! May be rename cms-content-cm.yaml into content-cm.yaml?